Installer/package.txt

```diff
@@ -527,6 +542,26 @@ component Runtime.iOS
 		file ios:iphoneos8.2/Release/template-ResourceRules.plist as "ResourceRules.plist"
 		file ios:iphoneos8.2/Release/Default-568h@2x.png as "Default4InchSplash.png" base ios:Default-568h@2x.png
 		file ios:iphoneos8.2/Release/fontmap as "fontmap"
+	into "[[ToolsFolder]]/Runtime/iOS/Device-8.3" place
+		executable ios:iphoneos8.3/Release/standalone-mobile-lib[[EditionTagLower]].lcext
```

Comment:
You should be able to base the 8.3 binaries on the 8.2 ones - it might make the package slightly smaller if its just minor API differences between the SDKs